roles/core/templates/sdcore-5g-values.yaml

```diff
@@ -323,20 +325,23 @@ omec-user-plane:
         subnet: {{ ran_subnet }} # Subnet for the gNB network
       access:
         iface: {{ core.data_iface }}
+        resourceName: "intel.com/intel_sriov_vfio_access"
```

Comment:
Did you test these changes deploying the UPF in `af_packet` mode? In other words, UPF gets properly deployed in `af_packet` mode after these changes are merged?

Reply:
yes, verified both DPDK/AF_PACKET mode successfully.